cpp/include/raft/linalg/distance_type.h

```diff
@@ -35,18 +35,18 @@ enum DistanceType : unsigned short {
   EucUnexpandedL2Sqrt = 5,
   /** basic inner product **/
   InnerProduct = 6,
-  /** ChebyChev (Linf) distance **/
-  ChebyChev = 7,
+  /** Chebyshev (Linf) distance **/
+  EucUnexpandedLinf = 7,
   /** Canberra distance **/
-  Canberra = 8
+  EucUnexpandedCanberra = 8,
```

Comment:
I'm surprised that Canberra is tagged as being `Euc` (Euclidean?). I would've thought that Euclidean only applies to root of square distance?

Reply:
I thought the same about many of the other distances as well (e.g. L1) and I'm just following the convention already established here. I can change it, but I'd prefer it be consistent (for L1 to be changed). I'm assuming Euclidean in this context to just be derived from the generalized Lp.

Reply:
After offline discussion, opened https://github.com/rapidsai/raft/issues/118